Source/Scene/Scene.js

```diff
@@ -1310,6 +1456,15 @@ define([
 
             scene._globeDepth.executeCopyDepth(context, passState);
 
+            if (scene.debugShowGlobeDepth) {
```

Comment:
Basically this function would have:
```javascript
var globeDepth = scene.debugShowGlobeDepth ? scene._debugGlobeDepths[i] : scene._globeDepth;
// now use globeDepth throughout instead of scene.__globeDepth
```
It will be a bit more, but that is the idea.